patches/minecraft/net/minecraft/world/entity/player/Player.java.patch

```diff
@@ -135,15 +135,21 @@
           if (!this.m_9236_().f_46443_) {
              this.m_36246_(Stats.f_12982_.m_12902_(this.f_20935_.m_41720_()));
           }
-@@ -865,6 +_,8 @@
+@@ -863,10 +_,13 @@
+          if (p_36383_ >= 3.0F) {
+             int i = 1 + Mth.m_14143_(p_36383_);
              InteractionHand interactionhand = this.m_7655_();
++            ItemStack currentItem = this.f_20935_;
```

Comment:
Think it's worth including a comment on the patch of why this is stored in a local variable?

Reply:
Ok, I'll add the comment tomorrow